hiera.yaml

```diff
@@ -0,0 +1,12 @@
+---
+version: 5
+
+defaults:
+  datadir: hieradata
```

Comment:
I'm just wondering here if the `datadir` should be `hiera_data` ... or is there any convention for the hiera data folder name? 🤔

Reply:
Could probably just use what is used in the skeleton control-repo.

https://github.com/puppetlabs/control-repo/blob/production/hiera.yaml

Reply:
Thanks for the reference to the control-repo!